js/plugin-loader.js

```diff
@@ -490,6 +490,15 @@ PluginLoader.prototype = {
   plugins: null,
   pluginMap: null,
   unresolvedImports: null,
+
+  _emitPluginEurekaInfo(pluginDef) {
```

Comment:
1. It doesn't look like "update eureka metedata" is an event at all. What happens here that we need to signal? When the plugin loader has loaded a plugin (either filesystem or dynamic) it already dispatches 'pluginLoaded'. I think even making it *know* that it's invoked by something-Eureka is a violation of modularity
2. This event  isn't even handled anywhere

Reply:
Yeah this may be dead code. There was more that was removed  from the original PR due to not being useful to the mediation layer than what actually was merged